stlab/forest.hpp

```diff
@@ -0,0 +1,1240 @@
+/*
+    Copyright 2013 Adobe
+    Distributed under the Boost Software License, Version 1.0.
+    (See accompanying file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+*/
+/**************************************************************************************************/
```

Comment:
Is there a reason, that the forest is in root of stlab and not in a folder like stlab/container/forest.hpp?

Reply:
Good question. It is at the top of ASL, so I assumed it'd be the same in stlab. @sean-parent do you have an opinion here?

Reply:
`forest` is at the same level where `copy_on_write` is currently. This would suggest that unless there's a reason to group headers together, they do not need to go into a directory. I'm going to leave the file where it is for now, but if you'd like to revisit the issue we certainly can.